frost-core/src/frost/keys/dkg.rs

```diff
@@ -10,23 +10,24 @@
 //! other participants with proof in zero knowledge, instantiated as a Schnorr signature,
 //! to protect against rogue-key attacks in the setting where `t ≥ n/2`.
 //!
-//! Pedersen's DKG is simply where each participant executes Feldman's
-//! Verifiable Secret Sharing (VSS) as the dealer in parallel, and derives their
-//! secret share as the sum of the shares received from each of the `n` VSS
-//! executions.
+//! In Pedersen's DKG, each of the `n` participants executes [Feldman's
+//! Verifiable Secret Sharing (VSS)][Feldman's VSS] as the dealer in parallel,
```

Comment:
Feldman's VSS is referenced twice here

Reply:
This is how to make some text a hyperlink to a named link in [rustdoc markdown](https://commonmark.org/help/) without making the name of the link the hyperlink text:

<img width="297" alt="image" src="https://github.com/ZcashFoundation/frost/assets/552961/ad6edb09-04fa-46ac-b899-654a70b94422">


<img width="556" alt="image" src="https://github.com/ZcashFoundation/frost/assets/552961/1741c53e-ce03-4be1-a9f6-136108c45914">